internal/store/organizations.go

```diff
@@ -48,11 +49,12 @@ type OrganizationDeleter interface {
 func (s *Store) Organizations(opts *atlas.OrganizationsListOptions) (interface{}, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		res := s.clientv2.OrganizationsApi.ListOrganizations(s.ctx)
+		params := new(atlasv2.ListOrganizationsApiParams)
 		if opts != nil {
-			res = res.Name(opts.Name).PageNum(opts.PageNum)
```

Comment:
are we excluding ItemsPerPage on purpose or we should set it?

Reply:
probably a bug, can you add it? but confirm the flag exists in the command

Reply:
it exists, added